jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
+    local -r local_gcs_stage_path="${WORKSPACE}/_output/gcs-stage"
+    KUBERNETES_RELEASE_URL="file://${local_gcs_stage_path}"
+    KUBERNETES_RELEASE=$(ls "${local_gcs_stage_path}" | grep ^v.*$)
+    if [[ -z "${KUBERNETES_RELEASE}" ]]; then
+      echo "FAIL! version not found in ${local_gcs_stage_path}"
+      return 1
+    fi
+}
+
+# Use a published version like "ci/latest" (default), "release/latest",
+# "release/latest-1", or "release/stable".
+# TODO(ixdy): maybe this should be in get-kube.sh?
+function set_release_vars_from_gcs() {
+    local -r published_version="${1}"
+    IFS='/' read -a varr <<< "${published_version}"
+    local -r path="${varr[0]}"
+    if [[ "${path}" == "release" ]]; then
+      local -r bucket="${KUBE_GCS_RELEASE_BUCKET}"
+    else
+      local -r bucket="${KUBE_GCS_DEV_RELEASE_BUCKET}"
+    fi
+    KUBERNETES_RELEASE=$(gsutil cat "gs://${bucket}/${published_version}.txt")
+    KUBERNETES_RELEASE_URL="https://storage.googleapis.com/${bucket}/${path}"
+}
+
+function set_release_vars_from_gke_cluster_version() {
+    local -r server_version="$(gcloud ${CMD_GROUP:-} container get-server-config --project=${PROJECT} --zone=${ZONE}  --format='value(defaultClusterVersion)')"
+    # Use latest build of the server version's branch for test files.
+    set_release_vars_from_gcs "ci/latest-${server_version:0:3}"
+}
+
+function call_get_kube() {
+    [[ "${JENKINS_USE_GET_KUBE_SCRIPT:-}" =~ ^[yY]$ ]] || exit 2
+    export KUBERNETES_RELEASE
+    export KUBERNETES_RELEASE_URL
+    KUBERNETES_SKIP_CONFIRM=y KUBERNETES_SKIP_CREATE_CLUSTER=y KUBERNETES_DOWNLOAD_TESTS=y \
+      "$(dirname "${0}")/get-kube.sh"
```

Comment:
Can we pass arguments into this program instead of relying on environment variables?